cookbook/sections/data-publishers.adoc

```diff
@@ -82,9 +82,137 @@ Note:
 
 Of course, always ensure your WNM is valid (see <<Validate a WIS2 Notification Message>> for more information).
 
-=== Validate a WMO Core Metadata Profile record
+=== Publishing a WIS2 Notification Message with embedded data
 
-The [pywcmp](https://github.com/wmo-im/pywcmp) tool provides a test suite to validate a message against the WCMP2 specification requirements, as well as a Python API for application integration.  Consult the pywcmp README on GitHub for more information/examples.
+A https://wmo-im.github.io/wis2-notification-message[WIS2 Notification Message] provides the ability to embed data as part of a message (see the https://wmo-im.github.io/wis2-notification-message/standard/wis2-notification-message-STABLE.html#_1_15_properties_content[Properties / Content] section of the specification).  A data publisher may wish to provide embedded data for direct access, thereby skipping the need for accessing the data via the notification link (see the https://wmo-im.github.io/wis2-notification-message/standard/wis2-notification-message-STABLE.html#_1_16_links[Links] section of the specification).  A data publisher also may not wish to manage or publish the data link to infrastructure to an HTTP or FTP server.
```

Comment:
A https://wmo-im.github.io/wis2-notification-message[WIS2 Notification Message] provides the ability to embed data as part of a message (see the https://wmo-im.github.io/wis2-notification-message/standard/wis2-notification-message-STABLE.html#_1_15_properties_content[Properties / Content] section of the specification). 

Providing embedded data for direct access, offers to users access to the data without the need for downloading the file via the notification link (see the https://wmo-im.github.io/wis2-notification-message/standard/wis2-notification-message-STABLE.html#_1_16_links[Links] section of the specification). 

In some cases, a data publisher also may not wish to manage or publish the data link to infrastructure to an HTTP or FTP server.